README mwmichelpach1.md

```diff
@@ -1,5 +1,12 @@
 # Cloudera Director Public Scripts
 
+## Contributing Code
+
+In order to contribute code you must sign the ICLA(https://github.com/cloudera/livy/wiki/Individual-Contributor-License-Agreement-(ICLA)(For independant Contributors on their own behalf)  or CCLA(https://github.com/cloudera/livy/wiki/Corporate-Contributor-License-Agreement-(CCLA)(If you are contributing for a company or orgonization).
```

Comment:
nit: have a space before the parentheses. So for example it should be "ICLA (https://". Same for CCLA.